config/database.yml

```diff
@@ -2,9 +2,9 @@ default: &default
   adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
-  username: <%= ENV['DB_USER'] %>
```

Comment:
Let's just set this in development, I really want to kill the defaults as much as I can in this file